python/tests/test_units.py

```diff
@@ -18,6 +18,8 @@ def test_variable_unit_repr(self):
         assert repr(var1.unit) == "\u212B" or repr(var1.unit) == "\u00C5"
         var1.unit = sc.units.m * sc.units.m
         assert repr(var1.unit) == "m^2"
+        var1.unit = sc.units.counts
+        assert repr(var1.unit) == "counts"
```

Comment:
Do we support `"count"` as input to `sc.Unit`?

Reply:
Yes, this just forwards to the upstream string parsing, which handles both singular and plural versions of base units.

Reply:
Then maybe add a test for that. You could just loop over a bunch of strings and test against `sc.Unit` and `repr(unit)`.